hypermocker/Cargo.toml

```diff
@@ -0,0 +1,17 @@
+[package]
```

Comment:
I want to get rid of Mockito, and use this crate instead. Once it can handle all Walkers' use cases, I will create a new repo for it.